SPECS/mysql/mysql.spec

```diff
@@ -54,7 +52,8 @@ make test
 %files
 %defattr(-,root,root)
 %license LICENSE
-%doc LICENSE  README router/LICENSE.router router/README.router
+%license LICENSE router/LICENSE.router
```

Comment:
@thcrain-msft Is it ok that there are two license sections? Is this expected?

Reply:
It is fine, even required to mention all files with license information.

---
In reply to: [517678162](https://github.com/microsoft/CBL-Mariner/pull/341#discussion_r517678162) [](ancestors = 517678162)

Reply:
It probably won't hurt any to leave it but I think you'll might as well remove the line that's just "%license LICENCE" since the file is now included in the new %license line

Reply:
Agreeing with Henry here. Also, the linter leaving two %license lines is a bug and will be fixed shortly.

Reply:
fixed!